assets/css/frontend-form/layout3.css

```
@@ -47,124 +49,130 @@
+.wpuf-form-add.wpuf-form-layout3 ul.wpuf-form li .wpuf-fields .wpuf-radio-list li,
+.wpuf-form-add.wpuf-form-layout3 ul.wpuf-form li .wpuf-fields .wpuf-checkbox-list li,
+.wpuf-form-add.wpuf-form-layout3 ul.wpuf-form li .wpuf-fields .wpuf-multi-select-list li {
+  margin-bottom: 12px;
 }
 .wpuf-form-add.wpuf-form-layout3 ul.wpuf-form li .wpuf-fields a.file-selector {
   display: block;
-  padding: 14px 0 14px 40px;
+  padding: 30px 20px;
   margin: 0;
   height: inherit;
   line-height: inherit;
```

Stylelint (17.14.0)

[error] Duplicate property "line-height" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)